src/features/products/components/Card/Card.tsx

```diff
@@ -26,12 +27,12 @@ const Card: FC<CardProps> = ({ product }) => {
   const oldPrice = getOldPrice(product.mainPrice, product.discount);
 
   const handleOpenProduct = () => {
-    navigate(`/products/${product.id}`);
+    navigate(`/${ERoutes.PRODUCTS_LIST}/${product.id}`);
   };
 
   return (
     <li className="card">
-      <Link to={`/products/${product.id}`}>
+      <Link to={`/${ERoutes.PRODUCTS_LIST}/${product.id}`}>
```

Comment:
Remove duplicates

Reply:
Remove Links. Use onClick navigate for img, titile, button